content/community/ssh-telegram.md

```diff
@@ -0,0 +1,9 @@
++++
```

Comment:
Why have /ssh-telegram.md AND /lup-telegram.md ?
They seem functionally equivalent to me and are more of a `network` comms_type, no?

Reply:
Ya, telegram is a network communication but I needed a test case for being able to group more than one communication together. So, I just made up fake lup/ssh telegram things :upside_down_face: